### DISCUSSION TITLE
Pimcore v10.4.3 Not able to search Assets Metadata

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#12438] [Q&A / Support] [answered]
Hello @dvesh3 @fashxp 

I can not able to Search Assets metadata on the Assets List view on Pimcore v10.4.3. 
https://demo.pimcore.fun/admin/

Pim v10.4.3
![image](https://user-images.githubusercontent.com/52573912/174526310-823cf94e-e74f-4d34-917f-723009ea1b59.png)

Pim v6.8.4
![image](https://user-images.githubusercontent.com/52573912/174526401-733e9ad9-32c4-43d7-9c55-68b366b1e30b.png)

Earlier we use the Pimcore v6.8.4 and I can search the Metadata.

So is there any configuration or issue, pls guide.

**Comment by dvesh3** [comment, marked as answer]
Always return empty result. Seems like a bug to me https://github.com/pimcore/pimcore/issues/12440

**Reply by kingjia90** [reply]
Pushed a PR https://github.com/pimcore/pimcore/pull/12447 to fix it.
It seems caused by this quoteIndentifier in https://github.com/pimcore/pimcore/pull/11950, so confirming that is happening since 10.3.6
```
}else{
   $filterField = $db->quoteIdentifier($filterField);
}
```
